tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -1726,3 +1726,291 @@ def test_parse_tribal_section_4_bad_quarter(tribal_section_4_bad_quarter, dfs):
         "representing the Calendar Year and Quarter formatted as YYYYQ"
 
     Tribal_TANF_T7.objects.count() == 0
+
+
+@pytest.fixture
+def t3_cat2_invalid_citizenship_file():
```

Comment:
Can we move these test to a separate file? Also can move the fixtures to conftest: see [this](https://docs.pytest.org/en/7.1.x/reference/fixtures.html)

Reply:
done, moved the T fixtures because the other tests ended up getting removed